src/util-lua-builtins.c

```diff
@@ -45,6 +46,7 @@ static const luaL_Reg builtins[] = {
     { "suricata.packet", LuaLoadPacketLib },
     { "suricata.rule", SCLuaLoadRuleLib },
     { "suricata.ssh", SCLuaLoadSshLib },
+    { "suricata.ja3", SCLuaLoadJa3Lib },
```

Comment:
nit: Jason told me we keep alphabetical order here.

Reply:
ok